src/triangulum/utils.clj

```diff
@@ -1,18 +1,40 @@
 (ns triangulum.utils
-  (:import java.io.ByteArrayOutputStream)
+  (:import java.io.ByteArrayOutputStream
+           java.util.Date)
   (:require [cognitect.transit  :as transit]
             [clojure.java.shell :as sh]
             [clojure.string     :as str]
             [clojure.set        :as set]
             [clojure.data.json  :as json]))
 
+;;==============================================================================
 ;; Text parsing
+;;==============================================================================
 
 (defn kebab->snake
-  "kebab-str -> snake_str"
+  "Converts kebab-str to snake_str."
   [kebab-str]
   (str/replace kebab-str "-" "_"))
 
+(defn kebab->camel
```

Comment:
There is a great library for this. https://github.com/clj-commons/camel-snake-kebab

Reply:
Thanks for pointing out this library, Matt! I don't think we need to import a new library for just these two functions, but it's good to know that this library exists in case we need to expand upon this functionality down the line.